scss/main/_item.scss

```diff
@@ -434,220 +373,184 @@
+                                background-color: #fff;
+                                border-color: #ddd;
+                                .dropdown-btn {
+                                    position: relative;
+                                    width: 30px;
+                                    height: 20px;
+                                    margin: 0 auto;
+                                    background-color: $yellow3;
+                                    border-bottom-left-radius: 6px;
+                                    border-bottom-right-radius: 6px;
+                                    &:hover {
+                                        background-color: $yellow2;
+                                    }
+                                    .arrow {
+                                        border-left: 6px solid transparent;
+                                        border-right: 6px solid transparent;
+                                        @include centering();
+                                        &--down {
+                                            border-top: 10px solid #fff;
+                                        }
+                                        &--up {
+                                            border-bottom: 10px solid #fff;
+                                        }
+                                    }
+                                }
                             }
-
-                            .arrow {
-                              border-left: 6px solid transparent;
-                              border-right: 6px solid transparent;
-                              @include centering();
-
-                              &--down {
-                                border-top: 10px solid #fff;
-                              }
-                              &--up {
-                                border-bottom: 10px solid #fff;
-
-                              }
+                            .panel-default>.panel-heading+.panel-collapse>.panel-body {
+                                border-top-color: #ddd;
                             }
-                          }
                         }
-
-                        .panel-default > .panel-heading + .panel-collapse > .panel-body {
-                          border-top-color: #ddd;
-                        }
-                      }
                     }
                 }
                 .header-wrapper {
-                  position: relative;
-                  padding: 26px 0;
-
-                  .en & {
-                    margin-left: 26px;
-                  }
-
-                  .he & {
-                    margin-right: 26px;
-                  }
-
+                    position: relative;
+                    padding: 26px 0;
+                    .en & {
+                        margin-left: 26px;
+                    }
+                    .he & {
+                        margin-right: 26px;
+                    }
```

Comment:
lot's of changes here - not sure if they are actual changes or just whitespace - need to make sure it's only whitespace changes
if it's actual design changes - need to document it